content/docs/reference/pkg/akamai/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: "akamai"
-title_tag: "Package akamai"
+title: "Akamai"
+title_tag: "Package Akamai"
 meta_desc: "A Pulumi package for creating and managing akamai cloud resources."
```

Comment:
```suggestion
meta_desc: "A Pulumi package for creating and managing Akamai cloud resources."
```

Reply:
I need to check the codegen for this - this was created by that

Reply:
Yeah, I wasn't sure if it was possible to hand author this, or if the change needs to be made upstream.